organizations/v0/views.py

```diff
@@ -0,0 +1,52 @@
+# pylint: disable=too-many-ancestors
+"""
+Views are listeners connected to the Django HTTP request
+pipeline.  When they observe a corresponding URL request they run the
+code contained within their definition.
+
+In this particular application, the views simply hand-off to the
+orchestration layer, which manages the application's workflows.
+"""
+from rest_framework import viewsets
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.permissions import IsAuthenticated
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+
+from organizations.models import Organization
+from organizations.serializers import OrganizationSerializer
+
+
+class OrganizationsViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+
+    **Use Cases**
```

Comment:
This would make since if you were using a single `APIView`. Given that you are using a `ViewSet`, it doesn't make much sense. At this stage, most folks in our codebase should be familiar enough with DRF that you don't really need anything beyond a minimal docstring.